NineChronicles.Headless/GraphTypes/NodeStatus.cs

```diff
@@ -64,8 +64,36 @@ public NodeStatusType()
                 });
             Field<ListGraphType<TxIdType>>(
                 name: "stagedTxIds",
+                arguments: new QueryArguments(
+                    new QueryArgument<AddressType>
+                    {
+                        Name = "address",
+                        Description = "Target address to query"
+                    }
+                ),
                 description: "Staged TxIds from the current node.",
-                resolve: context => context.Source.BlockChain.GetStagedTransactionIds()
+                resolve: context =>
+                {
+                    if (!context.HasArgument("address"))
+                    {
+                        return context.Source.BlockChain.GetStagedTransactionIds();
+                    }
+                    else
+                    {
+                        Address address = context.GetArgument<Address>("address");
+                        var stagedTransactionIds = context.Source.BlockChain.GetStagedTransactionIds();
```

Comment:
```suggestion
                        IImmutableSet<TxId> stagedTransactionIds = context.Source.BlockChain.GetStagedTransactionIds();
```

Reply:
@longfin Should I add references for these types? I tried to declare them before but they couldn't build 😢 

![Untitled](https://user-images.githubusercontent.com/42176649/107315240-a0040e80-6ad9-11eb-8bf2-25a157259497.png)

Reply:
It's from `System.Collections.Immutable`.

[see also](https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.iimmutableset-1?view=net-5.0&viewFallbackFrom=netstandard-2.1)